app/components/phone_input_component.html.erb

```diff
@@ -40,10 +41,21 @@
         as: :tel,
         required: required,
         label: false,
+        wrapper_html: {
+          class: 'margin-bottom-0',
+        },
         input_html: {
-          aria: { invalid: false },
+          aria: {
+            invalid: false,
+            describedby: "phone-input-error-#{unique_id}",
```

Comment:
Will need to rethink this `aria-describedby`, because apparently it honors descriptions which are visually hidden with `display: none`, which is how we've controlled the validation messages. Currently, it would announce every error message, regardless whether it's "active".

>by design, hiding the content (using CSS display:none or visibility:hidden or the HTML hidden attribute) of the element(s) referenced by these attributes does not stop the content from being used to provide the name/description.

https://www.tpgi.com/short-note-on-aria-labelledby-and-aria-describedby/

Reply:
It might be worth revisiting the decision to add individual elements for every error variant to the static markup of the page, and instead create them dynamically in the JavaScript. This would probably make things a little easier to manage anyways.

Reply:
This was revised in 432d152 to introduce a new component which dynamically adds error messages based on validity state.